test_conformance/events/action_classes.cpp

```diff
@@ -101,25 +111,25 @@ cl_int    Action::IGetPreferredImageSize3D( cl_device_id device, size_t &outWidt
         outDepth = maxDepth;
     }
 
-    outWidth /=2;
-    outHeight /=2;
-    outDepth /=2;
+    outWidth /= 2;
+    outHeight /= 2;
+    outDepth /= 2;
 
-    if (outWidth > 512)
-        outWidth = 512;
-    if (outHeight > 512)
-        outHeight = 512;
-    if (outDepth > 512)
-        outDepth = 512;
-    log_info("\tImage size: %d x %d x %d (%gMB)\n", (int)outWidth, (int)outHeight, (int)outDepth,
-             (double)((int)outWidth*(int)outHeight*(int)outDepth*4)/(1024.0*1024.0));
+    if (outWidth > 512) outWidth = 512;
+    if (outHeight > 512) outHeight = 512;
+    if (outDepth > 512) outDepth = 512;
+    log_info("\tImage size: %d x %d x %d (%gMB)\n", (int)outWidth,
+             (int)outHeight, (int)outDepth,
+             (double)((int)outWidth * (int)outHeight * (int)outDepth * 4)
+                 / (1024.0 * 1024.0));
 
     return CL_SUCCESS;
 }
 
-#pragma mark -------------------- Execution Sub-Classes -------------------------
+#pragma mark-------------------- Execution Sub-Classes -------------------------
```

Comment:
This change doesn't look right; occurs in other places in this PR too.

Reply:
Thanks - I have fixed this.

We should probably remove these #pragmas, as they're Xcode-specific and do nothing to improve the organization of the code for those not using these IDEs.  Indeed, I would more generally say that pragmas are not the proper solution for organizing code so that it is easy to search and navigate.

However, it seems best to leave their removal for another change, and leave this PR purely as a reformatting change.